app/parse.py

```diff
@@ -8,9 +28,59 @@
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def get_single_quote(quote_html: Tag) -> Quote:
+    return Quote(
+        text=quote_html.select_one(".text").text,
+        author=quote_html.select_one(".author").text,
+        tags=[tag.text for tag in quote_html.select(".tag")]
+    )
+
+
+def get_single_page_quotes(page: str) -> list[Quote]:
+    logging.info("Parsing quotes for a page")
+    soup = BeautifulSoup(page, "html.parser")
+    quotes_html = soup.select(".quote")
+    quotes = [get_single_quote(quote) for quote in quotes_html]
+    return quotes
+
+
+def get_next_page_url(page: str) -> str:
+    soup = BeautifulSoup(page, "html.parser")
+    next_html = soup.select_one(".next > a")
+    if next_html:
+        return next_html["href"]
```

Comment:
Consider using `urljoin(BASE_URL, next_html['href'])` to return the full URL instead of just the relative path. This will ensure that the `get_quotes` function works correctly when constructing the next page URL.